dev/breeze/src/airflow_breeze/commands/testing_commands.py

```diff
@@ -180,6 +190,21 @@ def _run_test(
         "--rm",
         "airflow",
     ]
+    run_cmd.extend(
```

Comment:
Pytest arguments are all generated (and unit tested in breeze's Python code - way better than it was in the  past in `entrypoint_ci.sh`